spec/migrations/20160708182331_remove_replication_worker_settings_spec.rb

```diff
@@ -0,0 +1,25 @@
+require_migration
+
+describe RemoveReplicationWorkerSettings do
+  let(:settings_change_stub) { migration_stub(:SettingsChange) }
+
+  migration_context :up do
+    it "removes the replication worker settings" do
+      settings_change_stub.create!(
+        :resource_type => "MiqServer",
+        :key           => "/workers/worker_base/replication_worker/memory_threshold",
+        :value         => "5.gigabytes"
+      )
+
+      settings_change_stub.create!(
+        :resource_type => "MiqServer",
+        :key           => "/workers/worker_base/replication_worker/replication/destination",
+        :value         => {:host => "somehost.example.com", :user => "root"}
```

Comment:
I don't think it would be stored this way.  I would expect two separate keys "/workers/worker_base/replication_worker/replication/destination/host" and "/workers/worker_base/replication_worker/replication/destination/user"